content/news/release-notes.adoc

```diff
@@ -18,6 +47,17 @@ Kiali 1.15 is installed with:
 
 '''
 
+=== 1.15.2
+_Fix Release: TBD_
```

Comment:
What is this TBD?

Reply:
oh, its because we didn't release 1.15.2. Are we merging this like this? or do we wait until 1.15.2 is released and we can update this?

Reply:
removed into separate PR